assets/src/edit-story/components/canvas/carousel/index.js

```diff
@@ -34,11 +33,12 @@ import { LeftArrow, RightArrow, GridView as GridViewButton } from '../../button'
 import Modal from '../../modal';
 import GridView from '../gridview';
 import DraggablePage from '../draggablePage';
+import useResizeEffect from '../../../utils/useResizeEffect';
 
 // @todo: Make responsive. Blocked on the header reimplementation and
 // responsive "page" size.
-const PAGE_HEIGHT = 50;
-const PAGE_WIDTH = PAGE_HEIGHT * 9 / 16;
+const PAGE_THUMB_HEIGHT = 50;
```

Comment:
Will this not be responsive?

Reply:
Already understood that thumbs will be handled separately, resolving.

Reply:
Not in this PR. I simply renamed the constant to avoid bumping into it all the time during refactoring.